### ISSUE TITLE
Adding rear-side of DFCs to predefined 'tokens' slot

### ISSUE THREAD

**Issue by ideocl4st** [Cockatrice/Cockatrice#804]
tl;dr : We should be able to add rear-side of DFCs to predefined 'tokens' slot in decklists.

I rarely used a DFC properly in Cockatrice - instead of properly transforming the card (Flip the card, create a token with name of the rear-side, etc...), I just flipped it over to back-side and stuck some attributes on it and called it transformed.

However, now that we are about to get DFCs that will turn into Planeswalkers, I think we need a easier way to create an object that displays the rear-side of a DFC.

I think allowing rear-side of the DFCs to be put into tokens slot so that they can be easily created with a few clicks (instead of having to type out the name) is the easiest solution that will not cause too much trouble for devs.

What do you guys think?

**Comment by ZeldaZach** [comment]
This is a dupe of #77 and we really should look into fixing this

**Comment by ideocl4st** [comment]
@ZeldaZach Oh, oops. I only searched for Double-Face Cards and DFCs. Feel free to close this one but yeah this will need to be taken care of...

**Comment by ZeldaZach** [comment]
Nah, it's fine. I think a 2nd ticket might be an incentive for someone to come up with an idea x)
